jaxrs/src/main/java/se/fortnox/reactivewizard/jaxrs/JaxRsResultSerializerFactory.java

```diff
@@ -0,0 +1,90 @@
+package se.fortnox.reactivewizard.jaxrs;
```

Comment:
Perhaps put this class in the `response` package with the other associated classes?